service/db.hpp

```diff
@@ -80,370 +93,309 @@ namespace raven
         }
     }
 
-    config_create_answer_db config_create(const json::json &config_create_data) noexcept
+    config_create_result config_create(const std::string &name) noexcept
     {
+        /*
+         * Create a config with the name passed as parameter
+         *
+         * Return the resulting config ID as well as the config key and read-only config key
+         *
+         * In case an error occur, the state will be set accordingly (`sql_error`, `fatal_error`)
+         *
+         */
+
         LOG_SCOPE_F(INFO, __PRETTY_FUNCTION__);
         using namespace std::string_literals;
         config_key_st config_key;
         config_key_st readonly_config_key;
-        config_id_st request_state_value;
-        std::optional<request_state> state_error = std::nullopt;
-        for (nb_tries_ = 0; nb_tries_ < maximum_retries_; ++nb_tries_) {
+        config_id_st config_id;
+        for (unsigned int nb_tries_ = 0; nb_tries_ < maximum_retries_; ++nb_tries_) {
             try {
                 json::json data_to_bind;
-                data_to_bind[config_name_keyword] = config_create_data[config_name_keyword];
-                data_to_bind["SETTINGS"] = json::json::object();
-                data_to_bind["OTHER_CONFIG"] = json::json::array();
+                data_to_bind[config_name_keyword] = name;
+                data_to_bind[config_settings_field_keyword] = json::json::object();
+                data_to_bind[config_includes_field_keyword] = json::json::array();
+                DLOG_F(INFO, "attempt nb: %i", nb_tries_);
```

Comment:
```suggestion
                DLOG_F(INFO, "attempt nb: %u", nb_tries_);
```
Since `nb_tries_` is unsigned.